libs/core/src/lib/action-sheet/action-sheet-item/action-sheet-item.component.html

```diff
@@ -0,0 +1,8 @@
+<li>
+    <button fd-button [class]="negative ? 'fd-button--negative' : 'fd-button--transparent fd-button--text-alignment-left'"
+            [class.fd-button--compact]="compact"
+            [class.fd-button--full-width]="mobile"
```

Comment:
This class should be used everytime, because it looks like that
![image](https://user-images.githubusercontent.com/26483208/95832838-23377280-0d3b-11eb-9f8b-5794e8970610.png)